contrib/localnet/preparams/zetaclient-dry.json

```diff
@@ -0,0 +1 @@
7195983583463713035873299160921004565254893237117131879786832408136722040717983131275408386456920384917380232351721007855708138535029096408229070315382667384133144023182833198443764766682690532886824751001859544330542870823512696556347173113873437686605831707256350994689481256583767447820115245376602495367798324074223225243671987502671118041237532180116019834429479650042237727165792113888482653902202712312976013890261549214358648587553687758535859263479105326312998254128290399923084611766459095,"H2i":20267809306051929473755357295460839897817576960897369326358391853878323580763031939703416355971967787898877573278961126880982846461638834221440160340323710140120316181532600229569341187515171607208297747517884384860372439732323247414674600630502312226506572200779791210288097651976034413667395951806984985856410377513680698445659810243668766096144023886006682304042976671059049694630181249249086203534825129095399138300747685224105095900977529727863173451740454984673210860242916343968748751800869334123207867924817967903482122599935520439891918414463303618255593391173249677201084158287527731249038886214575429914925,"Alpha":15232288501155126414174293747948632915711049881435983179248971803818961872131050268297152667746815474235059270566151869937613226205904042058632884546405243279776357283423396444949858106030059349790417734747658603948847253772074200505590645825448503491091204853290205639265689360330263453551656983417049695348363337806932706115832151082219380350164844683885899464466226595982091401679106039451907722079995941564575664045182045711957375117742281734764755138550943760771202630513886182062408886087763171411170854616354415307869343564751981078237553163684447568719140654334566890985915688544223802746957249330468312659177,"Beta":4353498303700459848238482697559980851773260090218351412879925363985514278702042517652850621038607541710984758258130834163317044056981758606007148239364594436998716261836758115845266570142808259895860353221019706804879590530318230298407442402951649319876191837746453723294041114182887647763946772174817233877015781237868268825073373099698589188993796485768148991765003278177874473208568781795646460891808118023860972001039516739613843087523805038056884787520501544922122419148955386357242788870033121988755592501380136444721243712395841163877803720185872032713712205066325715919614305534706069841648471471402320727198,"P":87133384790273113244147927852819878017648452028464891712850238445366418598289687727182310841569919608199042998143060083308022847830934895041151800229507257356019217970676820515712163859796763117033506599017918812578119583775726313194230372401246384191616720422271486005398218809393467006851501738239256531279,"Q":69117382905446951169716227724565438057852384568038786220479380096171332400324758564030124041668699999791922291722081124274579949459364583318711009863456327022203560779211251614199475812292745808431973962874719723749804255016830760458306531203399084244402034196999873094856726066535005997886892300562080649181}
```

Comment:
can we add to PR description what this file is for more context?

also there is file named `2` in diff, not sure if this is mistake or needed?